.github/workflows/verify.yaml

```diff
@@ -0,0 +1,23 @@
+name: Verify
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - main
+      - master
+
+jobs:
+  verify:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go: [ '1.17', '1.18', '1.19' ]
+    name: Go ${{ matrix.go }} sample
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup go
+        uses: actions/setup-go@v3
+        with:
+          go-version: ${{ matrix.go }}
+      - run: go run cmd/main.go verify
```

Comment:
We need some Truth in Advertising here to indicate that the Landscape YAML file is the object of verification. 
Could use the name of the Action on line 1 and/or move main.go to a verify-landscape folder or review the name of the command passed to main.go

Reply:
Actually I think going back to the earlier comment of "not using the flag", updating this (which I'll do and resolve) should make that clear.